README.md

```diff
@@ -90,7 +90,7 @@ image we provide:
 docker run
   -e MARGE_AUTH_TOKEN="$(cat marge-bot.token)" \
   -e MARGE_SSH_KEY="$(cat marge-bot-ssh-key)" \
-  smarketshq/marge-bot:0.1.0 \
+  smarketshq/marge-bot \
```

Comment:
That makes sense when you are running the image you built yourself. I expect most people will want to use the latest image on docker hub and we should bump `latest` there as well.